pylti/tests/test_flask_app.py

```diff
@@ -1,32 +1,50 @@
+"""
+Test pylti/test_flask_app.py module
+"""
 from flask import Flask, session
 
 from pylti.flask import lti
 from pylti.common import LTI_SESSION_KEY
 
-
+# pylint: disable=invalid-name
 app = Flask(__name__)
 
 
 class ExceptionHandler(object):
+    """
+    Custom exception handler.
+    """
     exception = None
 
     def set(self, exception):
+        """
+        Setter: set class variable exception.
+        """
         self.exception = exception
 
     def get(self):
+        """
+        Return exception if not None otherwise returns None
+        """
         if self.exception is None:
             return None
         else:
             return self.exception['exception']
 
     def reset(self):
+        """
+        Reset variable exception
+        """
         self.exception = None
 
-app_exception = ExceptionHandler()
```

Comment:
I would actually just do the invalid-name exception here since a function really isn't a constant